SIL.BuildTasks/StampAssemblies/StampAssemblies.cs

```diff
@@ -152,38 +190,64 @@ public VersionParts GetExistingAssemblyFileVersion(string contents)
 			return GetExistingAssemblyVersion("AssemblyFileVersion", contents);
 		}
 
-		public VersionParts ParseVersionString(string contents, bool allowHashAsRevision = false)
+		public static VersionParts ParseVersionString(string contents, VersionFormat format = VersionFormat.File)
```

Comment:
API change here, Are we going to 5.0 for the library now?

Reply:
I think this API was only public for unit test purposes. If projects are using it, I can keep the old API as well.